### PR TITLE
mlkem release prep

### PR DESCRIPTION
**Pull request by franziskuskiefer**

### CODE REVIEW

File: libcrux-intrinsics/Cargo.toml [hunk 1]
```diff
@@ -9,12 +9,6 @@ repository.workspace = true
 readme.workspace = true
```
Comment (jschneider-bensch) [inline libcrux-intrinsics/Cargo.toml:10]:
```suggestion
exclude = [ "/proofs" ]

```
We want to exclude proofs from the published crates, also on the other ones.

File: libcrux-ml-kem/Cargo.toml
Comment (jschneider-bensch) [inline libcrux-ml-kem/Cargo.toml:1]: Why do we want to exclude the `/tests` directory?

**Review by jschneider-bensch — CHANGES_REQUESTED** [review]
This looks good to me, but we don't want to publish the `proof` directories with the crates.

File: libcrux-ml-kem/Cargo.toml
Comment (franziskuskiefer) [reply libcrux-ml-kem/Cargo.toml:1]: It's huge, and I'm not sure people really use the tests of a published crate.

**Review by jschneider-bensch — APPROVED** [review]
Thanks, that resolves my questions.

**Pull request merged**